main/networking/Runtime.ts

```diff
@@ -31,6 +31,7 @@ enum MsgType {
   CHALLENGE_DATA,
   LOG,
   DEVICE_DATA,
+  INPUTS = 6
```

Comment:
Think you should add all the `= <num>` now that it skips a value

Reply:
Alright

Reply:
Done